server/api/auth/createUserWithIdp.js

```diff
@@ -45,9 +45,14 @@ module.exports = (req, res) => {
 
   const { idpToken, idpId, ...rest } = req.body;
 
+  console.log('Req.body', req.body);
   // Choose the idpClientId based on which authentication method is used.
   const idpClientId =
-    idpId === FACEBOOK_IDP_ID ? FACBOOK_APP_ID : idpId === GOOGLE_IDP_ID ? GOOGLE_CLIENT_ID : null;
+    idpId === FACEBOOK_IDP_ID
+      ? FACBOOK_APP_ID
+      : idpId === GOOGLE_IDP_ID
+      ? GOOGLE_CLIENT_ID
+      : 'client-id';
```

Comment:
If this 'client-id' string is meaningful variable for generic OpenId setup, maybe it should also be a constant with name that better communicates that it is.

Reply:
This was just for testing. I thought I removed the WIP commit with hard-coded values from the PR but it seems I forgot to push that change. In real life, you should do a similar check for your custom implementation e.g. LinkedIn like you do e.g. for Google, and add also the required constants.